content/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,269 @@
+
+* **k8s-manifests-deploy-private.yaml**, including the following 3 *Deployment* manifests:
+
+  * *emailservice*;
+  * *paymentservice*;
+  * *shippingservice*;
+* **k8s-manifests-deploy-public.yaml**, including the following 7 *Deployment* manifests:
+
+  * *frontend*;
+  * *recommendationservice*;
+  * *productcatalogservice*;
+  * *checkoutservice*;
+  * *cartservice*;
+  * *currencyservice*;
+  * *redis-cart*;
+  * *adservice*;
+* **k8s-manifests-service-public.yaml**, including the following 2 *Service* manifests:
+
+  * *frontend*;
+  * *frontend-external*;
+
+### Deploy Application Microservices to AWS EKS Cluster
+
+C﻿reate the namespace *boutique* in the cluster and then deploy 7 *Deployment* and 2 *Service* resources to the namespace:
+
+```markdown
+$ kubectl create ns boutique
+$ kubectl config set-context --current --namespace boutique
+$ kubectl apply -f k8s-manifests-deploy-public.yaml                  
+deployment.apps/recommendationservice created                                                  
+deployment.apps/frontend created                                                             
+deployment.apps/productcatalogservice created                                               
+deployment.apps/checkoutservice created                                                      
+deployment.apps/cartservice created                                               
+deployment.apps/currencyservice created                                                
+deployment.apps/redis-cart created                                                      
+deployment.apps/adservice created                                                            
+
+$ kubectl apply -f k8s-manifests-service-public.yaml           
+service/frontend created                                                              
+service/frontend-external created    
+
+$ kubectl get svc
+frontend                ClusterIP      172.20.103.129   <none>                                                                    80/TCP                            40s   <none>
+frontend-external       LoadBalancer   172.20.16.223    a52d7c861c01c4466803a44373bc11dc-1387384363.us-east-2.elb.amazonaws.com   80:31482/TCP                      40s   <none>                                    
+```
+
+### Deploy Application Microservices to PCE Private Cluster
+
+S﻿imilarly, create the namespace *boutique* in the cluster and then deploy 3 *Deployment* resources to the namespace:
+
+```markdown
+$ kubectl create ns boutique
+$ kubectl config set-context --current --namespace boutique
+$ kubectl apply -f k8s-manifests-deploy-private.yaml    
+deployment.apps/emailservice created
+deployment.apps/paymentservice created
+deployment.apps/shippingservice created
+```
+
+### Deploy Virtual Application Network
+
+Define the Virtual Application Network using *Skupper* on both AWS public and PCE private clusters:
+
+#### 1. In AWS public cluster, deploy the *aws-public* application router.
```

Comment:
I've added the environment setup details in the section Prerequisites for both kubectl and Skupper for accessing the public and private K8s clusters. 

In case kubectl already sets up the namespace, e.g., 'kubectl config set-context --current –namespace boutique', the skupper command will run in this namespace, no necessarily to specify again with --namespace.